qt/mainwindow.cpp

```diff
@@ -222,9 +222,13 @@ MainWindow::MainWindow(Framework & framework,
   RoutingSettings::LoadSession(m_pDrawWidget->GetFramework());
 }
 
-#if defined(Q_WS_WIN)
-bool MainWindow::winEvent(MSG * msg, long * result)
+#if defined(OMIM_OS_WINDOWS)
+bool MainWindow::nativeEvent(QByteArray const & eventType, void * message, qintptr * result)
 {
+  (void)eventType;
+  (void)result;
```

Comment:
Why are these needed?

Reply:
Silencing warnings about unreferenced parameters. Let me know if unnamed arguments are more preferred.

Reply:
Aren't they referenced below?

I personally prefer `/*message*/` instead, to save vertical lines space.